coordinator-tool/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 - Add `description` argument to `new-election`
+- Update election contract version
```

Comment:
The coordinator-tool was changed:
`Add option to use coordinator tool to verify the initial weights from a contract address`